extensions/amp-story-player/validator-amp-story-player.protoascii

```diff
@@ -42,4 +42,40 @@ descendant_tag_list {
   tag: "A"
   tag: "SPAN"
   tag: "I-AMPHTML-SIZER"  # Only allowed when document is transformed AMP
+  tag: "IMG"
+}
+tags: {
+  html_format: AMP
+  tag_name: "IMG"
+  spec_name: "AMP-STORY-PLAYER IMG"
+  mandatory_ancestor: "AMP-STORY-PLAYER"
+  attrs: {
+    name: "loading"
+    value: "lazy"
+  }
+  attrs: { name: "data-amp-story-player-poster-img" }
+  attrs: { name: "alt" }
+  attrs: { name: "attribution" }
+  attrs: { name: "border" }  # Not valid html5 but commonly used and supported.
+  attrs: {
+    name: "decoding"
+    value: "async"
+    value: "auto"
+    value: "sync"
+  }
+  attrs: { name: "height" }  # Not valid html5 but commonly used and supported.
+  attrs: { name: "ismap" }
+  attrs: {
+    name: "longdesc"
```

Comment:
Since this is a new usage, I'd like to propose moving away from longdesc. Previously IMG was only allowed in a NOSCRIPT context which is somewhat of a backwards compatibility mode. Longdesc is not supported by any major browser.